kinder/ci/tools/update-workflows/templates/workflows/dryrun-tasks.yaml

```diff
@@ -100,6 +100,7 @@ tasks:
   args:
     - -c
     - |
+      # TODO: "upgrade plan" does not have "--dry-run", yet. Add a "plan" call here, when it's supported.
```

Comment:
IIRC, upgrade plan doesn't apply any change to the cluster, what  does the dry-run for the plan suppose to be?

Reply:
good point.

our dry run has a fake API client, so i think it means we can fake the calls that "plan" does as well?
or we could say that there is no plan to add --dry-run to "plan" completely.